### ISSUE TITLE
[Material] RangeSlider should have Semantics nodes for each thumb, rather than having one for the whole RangeSlider

### ISSUE THREAD

**Issue by clocksmith** [flutter/flutter#60715]
Currently, the `RangeSlider` uses a single set of `CustomSemanticsAction`s to control each thumb and announces the whole range. Instead, each thumb should be focusable and treated like a single Slider (seek control). Thumbs should not be able to pass each other, just like it works with gestures.

Related issue: https://github.com/flutter/flutter/issues/53958

**Comment by clocksmith** [comment]
This could help: https://master-api.flutter.dev/flutter/rendering/RenderObject/assembleSemanticsNode.html

RenderCustomPaint does something similar (but more complicated)
